scripts/release/generate-changelog.js

```diff
@@ -29,6 +28,8 @@ const INTERNAL_EMOJI_PRIORITY = [
   '⚗️', // experiment
 ]
 const EMOJI_REGEX = /^\p{Emoji_Presentation}/u
+const PACKAGES = ['rum', 'logs', 'rum-slim', 'rum-react', 'worker']
```

Comment:
suggestion: avoid hardcoding this list, as it is error prone and we [sometimes forget to update it](https://github.com/DataDog/browser-sdk/pull/2870).